docs/products/kafka/kafka-connect/howto/snowflake-sink.rst

```diff
@@ -0,0 +1,236 @@
+
+    * ``value.converter.schema.registry.url``: pointing to the Aiven for Apache Kafka schema registry URL in the form of ``https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT`` with the ``APACHE_KAFKA_HOST`` and ``SCHEMA_REGISTRY_PORT`` parameters :ref:`retrieved in the previous step <connect_sink_snowflake_prereq>`.
+    * ``value.converter.basic.auth.credentials.source``: to the value ``USER_INFO``, since you're going to login to the schema registry using username and password.
+    * ``value.converter.schema.registry.basic.auth.user.info``: passing the required schema registry credentials in the form of ``SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD`` with the ``SCHEMA_REGISTRY_USER`` and ``SCHEMA_REGISTRY_PASSWORD`` parameters :ref:`retrieved in the previous step <connect_sink_snowflake_prereq>`. 
+
+
+* ``snowflake.url.name``: The URL to access the Snowflake service
+* ``snowflake.user.name``: The connection user
+* ``snowflake.private.key``: The user's private key
+* ``snowflake.private.key.passphrase``: The private key passphrase
+* ``snowflake.database.name``: The Snowflake database name
+* ``snowflake.schema.name``: The Snowflake schema name
+
+
+Create a Kafka Connect connector with the Aiven Console
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, access the `Aiven Console <https://console.aiven.io/>`_ and select the Aiven for Apache Kafka® or Aiven for Apache Kafka Connect® service where the connector needs to be defined, then:
+
+1. Click on the **Connectors** tab
+2. Clink on **Create New Connector**, the button is enabled only for services :doc:`with Kafka Connect enabled <enable-connect>`.
+3. Select the **Snowflake Sink**
+4. Under the *Common* tab, locate the **Connector configuration** text box and click on **Edit**
+5. Paste the connector configuration (stored in the ``snowflake_sink.json`` file) in the form
+6. Click on **Apply**
+
+.. Note::
+
+    The Aiven Console parses the configuration file and fills the relevant UI fields. You can review the UI fields across the various tab and change them if necessary. The changes will be reflected in JSON format in the **Connector configuration** text box.
+
+7. After all the settings are correctly configured, click on **Create new connector**
+8. Verify the connector status under the **Connectors** tab
+9. Verify the presence of the data in the target Snowflake database
+
+.. Note::
+
+    Connectors can be created also using the dedicated :ref:`Aiven CLI command <avn_service_connector_create>`.
+
+
+Example: Create a Snowflake sink connector on a topic in Avro format
+--------------------------------------------------------------------
+
+The example creates an Snowflake sink connector with the following properties:
+
+* connector name: ``my_snowflake_sink``
+* source topics: ``test``
+* Snowflake database: ``testdb``
+* Snowflake schema: ``testschema``
+* Snowflake URL: ``https://XX0000.eu-central-1.snowflakecomputing.com:443``
```

Comment:
 In the end I used URL without https and the port. and it worked

Reply:
so, only `XX0000.eu-central-1.snowflakecomputing.com`?